modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1368,6 +1368,74 @@ public void testAddObjectActionWithMoreThanOneObjectEntry()
 		_objectActionLocalService.deleteObjectAction(objectAction3);
 	}
 
+	@Test
+	public void testAddObjectActionWithMoreThanOneUpdate() throws Exception {
```

Comment:
I don't think we are testing the addition of an object action thinking: 
I saw that we have this test method in this class: testConcurrentObjectActions()
So, I thought about `testSequentialObjectActions()` because, in the future, we could add more scenarios to this method.
What do you think? Do you have a suggestion?

Reply:
I liked the suggestion 😄 I updated the code.